### PR TITLE
Add ctrl-w o to Roadmap

### PR DESCRIPTION
**Pull request by max-sixty**
**Which issue(s) this PR fixes**

Fixes https://github.com/VSCodeVim/Vim/issues/4513

### CODE REVIEW

**Comment by max-sixty** [comment]
Separately, would you be open to renaming "Roadmap" to something like "Mappings & Roadmap"? It's the best (only?) source of keymappings in the docs. Having a single explicit place with all key mappings would be great for new users.

**Comment by J-Fields** [comment]
Thanks!

> Separately, would you be open to renaming "Roadmap" to something like "Mappings & Roadmap"? It's the best (only?) source of keymappings in the docs. Having a single explicit place with all key mappings would be great for new users.

Yeah, possibly. Maybe the README language needs to make it clearer that the roadmap includes all the supported mappings (or attempts to, any way)

File: ROADMAP.md [hunk 1]
```diff
@@ -524,6 +524,7 @@ Since the list is too long, now we just put those already supported options here
 | :white_check_mark: :star: | :vsp {file}       | Split vertically current window in two.                                 |                                                                                                                        |
 | :white_check_mark:        | &lt;ctrl-w&gt; s  | Split current window in two.                                            |                                                                                                                        |
 | :white_check_mark: :star: | &lt;ctrl-w&gt; v  | Split vertically current window in two.                                 |                                                                                                                        |
+| :white_check_mark: :star: | &lt;ctrl-w&gt; o  | Close other editor groups                                               |                                                                                                                        |
```
Comment (J-Fields) [inline ROADMAP.md]: Was about to merge, then realized that the other descriptions end in a period. Hate to be that guy, but could you add one?

Comment (max-sixty) [inline ROADMAP.md]:
```suggestion
| :white_check_mark: :star: | &lt;ctrl-w&gt; o  | Close other editor groups.                                              |                                                                                                                        |
```

Comment (max-sixty) [reply ROADMAP.md]: Do you squash on merge or do you need it on the same commit?

Comment (J-Fields) [reply ROADMAP.md]: Squash on merge, don't worry about it. Thanks 🙂

**Pull request merged**